test/bdd/fixtures/vc-rest/docker-compose.yml

```diff
@@ -61,6 +61,33 @@ services:
     networks:
       - couchdb_bdd_net
 
+  holder.vc.rest.example.com:
```

Comment:
we need to add this to sandbox and aws.
@fqutishat @rolsonquadras

Reply:
Yes, I'm going to update sandbox next